announcements.html

```diff
@@ -79,7 +82,9 @@ <h5>June 20, 2020</h5>
 		  <div class="timeline-content">
 			<h3>Lorem ipsum dolor</h3>
 			<hr class="mw-100" style="margin-top:.5rem; margin-bottom:.75rem; background-color:#f47621"/>
-			<p>Lorem ipsum dolor sit amet, quo ei simul congue exerci, ad nec admodum perfecto mnesarchum, vim ea mazim fierent detracto. Ea quis iuvaret expetendis his, te elit voluptua dignissim per, habeo iusto primis ea eam.</p>
+
+			<p class="description">Lorem ipsum dolor sit amet, quo ei simul congue exerci, ad nec admodum perfecto mnesarchum, vim ea mazim fierent detracto. Ea quis iuvaret expetendis his, te elit voluptua dignissim per, habeo iusto primis ea eam.</p>
```

Comment:
why the class description is added to it? @anushbhatia

Reply:
i did not added to it.

Reply:
While resolving it might have occurred